lib/utils/isStandardSyntaxUrl.cjs

```diff
@@ -8,7 +8,7 @@ const hasScssInterpolation = require('./hasScssInterpolation.cjs');
 const hasTplInterpolation = require('./hasTplInterpolation.cjs');
 
 const IS_LESS_VARIABLE_IN_URL = /^@@?[\w-]+$/;
-const IS_SCSS_VARIABLE_IN_URL = /^[$\s\w+\-,./*'"@]+$/;
+const IS_SCSS_VARIABLE_IN_URL = /^[$\s\w+\-,./*'"@#?]+$/;
```

Comment:
This regex is really lax but [`isScssVariable`](https://github.com/stylelint/stylelint/blob/main/lib/utils/isScssVariable.mjs) is even worse.
Since we are not officially supporting Sass, let's not bother ourselves and hack-fix it.